runtime/catalog-schema-registry/src/test/java/io/aklivity/zilla/runtime/catalog/schema/registry/internal/SchemaRegistryCatalogHandlerTest.java

```diff
@@ -36,6 +37,7 @@ public class SchemaRegistryCatalogHandlerTest
 {
     private SchemaRegistryCatalogConfig config;
     private EngineContext context = mock(EngineContext.class);
+    private EngineConfiguration configuration = mock(EngineConfiguration.class);
```

Comment:
We don't typically need to mock these in the tests, since they are simple implementations with no dependencies.